### DISCUSSION TITLE
Move discussions to different repo

### DISCUSSION THREAD

**Discussion by carlosmintfan** [community/community#138805] [Discussions] [answered]
### Select Topic Area

Question

### Body

Is it possible to move discussions from one repo to a different one in the same organization?

**Comment by LiteBrite82** [comment, marked as answer]
Hi @carlosmintfan 
An individual discussion, yes. The whole of a repos discussions at once, no. 
See [Transferring a Discussion](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#transferring-a-discussion)

I feel like it *could* be possible to create a GitHub Action to automate a bulk transfer; but that is past my current skill set. You could post a request for help with creating one in the `Actions` category. 

I checked the GitHub Actions Marketplace to see if one already existed for discussions; they had some super cool tools, but I didn't see one for bulk discussion transfers.

**Reply by carlosmintfan** [reply]
Hi, thank you for your answer. I only have a few discussions anyway, so transferring would not be a problem; however, as changing the preferred repo for organisation discussions would break links, I now prefer to keep it as it is.

**Comment by agriyakhetarpal** [comment]
To anyone who comes across a similar need, I found that someone has indeed created a bulk transfer tool for this in October 2025: https://josh-ops.com/posts/github-discussions-migration-utility.

Disclaimer: I have neither tried it personally, nor am I its author; and thus cannot provide guarantees of any kind.
